test/unit/inflector_test.rb

```diff
@@ -38,5 +38,33 @@ def setup
       assert_equal "thing", @inflector.pluralize("thing", 1)
       assert_equal "thing", @inflector.pluralize("things", 1)
     end
+
+    test "#apply_inflections will apply any custom inflections from file" do
+      file = "./test/fixtures/skeleton/custom_inflections.yml"
+      custom_inflector = custom_inflector_for(file)
+
+      expected_inflections = {
+        "acronym" => ["GraphQL", "MRuby", "TOS"],
+        "irregular" => [["analysis", "analyses"], ["reserve", "reserves"]],
+        "uncountable" => ["payment_details"],
+      }
+
+      assert_equal custom_inflector.inflections, expected_inflections
```

Comment:
I think this is testing the wrong thing. `#custom_inflector_for` goes to the trouble of creating a new `Packwerk::Inflector` and applying the custom inflections to it, but then the inflector gets thrown away and we’re just looking at the `@inflections` instance variable on the `Packwerk::Inflections::Custom` object to see what it contains. So we’re not testing whether those inflections have been applied to anything, only that they were successfully read from the file. We should either change the test description to reflect what it’s really doing or rewrite the test so that `#custom_inflector_for` actually returns the inflector and we try it out by seeing what e.g. `custom_inflector.underscore('GraphQL')` and `custom_inflector.singularize('payment_details')` return.

Reply:
Fixed in: [`cda9947` (#53)](https://github.com/Shopify/packwerk/pull/53/commits/cda994752819ae919689704b08351f8e5ddd72e7)